lumicks/pylake/force_calibration/tests/conftest.py

```diff
@@ -16,7 +17,7 @@ def lorentzian_filtered(f, fc, diffusion_constant, alpha, f_diode):
     def lorentzian_td(corner_frequency, diffusion_constant, alpha, f_diode, num_samples):
         f = np.arange(0, num_samples)
         power_spectrum = ReferenceModels.lorentzian_filtered(f, corner_frequency, diffusion_constant, alpha, f_diode)
-        data = np.fft.irfft(np.sqrt(np.abs(power_spectrum))) * (num_samples - 1) * 2
+        data = np.fft.irfft(np.sqrt(np.abs(power_spectrum * 0.5)), norm="forward")
```

Comment:
The `numpy` docs mention something about parts of `norm` only being available from `numpy` v1.20: https://numpy.org/doc/stable/reference/generated/numpy.fft.irfft.html

You might need to bump the minimum required version: https://github.com/lumicks/pylake/blob/667e7ff64ec8a415ade23f638e42fcb4dc3bd592/setup.py#L55

Reply:
Good catch! I will update the requirement.